signac_dashboard/dashboard.py

```diff
@@ -512,9 +540,13 @@ def favicon():
         self.add_url("views.home", ["/"])
         self.add_url("views.settings", ["/settings"])
         self.add_url("views.search", ["/search"])
+        self.add_url("views.project_info", ["/project/"])
```

Comment:
I can't decide if I like trailing slashes or not, but feels a tiny bit weird to have a trailing slash for `/jobs/` and `/project/` but not `/settings` and `/search` (and the internal endpoints for `/modules`).

Reply:
For someone who's picky about grammar and word usage, being consistent with the trailing slash isn't something I care about, surprisingly.